spotseeker_restclient/spotseeker.py

```diff
@@ -31,6 +31,7 @@ def search_spots(self, query_tuple):
 
         dao = SPOTSEEKER_DAO()
         url = "/api/v1/spot?" + urlencode(query_tuple)
+        print url
```

Comment:
Not planning on leaving this in for any reason, are we? This will cause mod wsgi failures under certain configurations.